app/attributes/attributecontroller.cpp

```diff
@@ -99,6 +99,17 @@ QgsAttributeEditorContainer *AttributeController::autoLayoutTabContainer() const
   return root.release();
 }
 
+bool AttributeController::evaluateHasTabs( QgsAttributeEditorContainer *container )
+{
+  bool hasTabs = !container->children().isEmpty();
+  for ( QgsAttributeEditorElement *element : container->children() )
+  {
+    QgsAttributeEditorContainer *elemContainer = static_cast<QgsAttributeEditorContainer *>( element );
```

Comment:
you need to first check element->type() == QgsAttributeEditorElement::AeTypeContainer before doing static cast